config/prod/prow/OWNERS

```diff
@@ -1,8 +1,10 @@
 approvers:
```

Comment:
I am not sure this file will do much in the near future. We are moving to CODEOWNERS.

Reply:
Yeah I can see that.. I'll close this PR then. Will try to bring it up again if needed after we move to CODEOWNERS.